shm/inc/cargo.hpp

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include <memory>
+#include <string>
+#include "observer.hpp"
+#include "time.hpp"
```

Comment:
I think that newline beetween standard library includes and project files would be nice.

Reply:
Indeed. Done.